content/docs/hooks-overview.md

```diff
@@ -128,9 +128,9 @@ function FriendStatus(props) {
 }
 ```
 
-In this example, React would unsubscribe from our `ChatAPI` when the component unmounts, as well as before re-running the effect due to a subsequent render. (If you want, there's a way to [tell React to skip re-subscribing](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects) if the `props.friend.id` we passed to `ChatAPI` didn’t change.)
+इस उदाहरण में, React हमारे `ChatAPI` से तब अनसब्सक्राइब हो जाती है जब कौम्पोनॅन्ट अनमाउंट हो जाता है, साथ ही बाद में रेंडर करने के कारण effect को फिर से चलाने से पहले। (यदि आप चाहते हैं, यहाँ एक तरीका है [React को री-सब्सक्राइब छोड़ना बताएं।](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects) अगर 'props.friend.id' जो हमने `ChatAPI` को पास किया है,नहीं बदलता तो।)
```

Comment:
`साथ ही बाद में रेंडर करने के कारण effect को फिर से चलाने से पहले` => `साथ ही बाद के रेंडर के वजह से effect को दुबारा चलने से पहले`

how about this?

Reply:
You translated `effect` as well 😄 . Rest everything looks good.

Reply:
`चलने` ---> `चलाने`...what do you think?

Reply:
`बाद के` ---? `अगले`...what do you think?

Reply:
`साथ ही अगले रेंडर के वजह से effect को दुबारा चलाने से पहले` --> this would be the final sentence after the change I mentioned above.

Reply:
Minor change in this 
`के वजह` => `की वजह`

rest looks good